qmail-remote.c

```diff
@@ -47,6 +53,13 @@ saa reciplist = {0};
 
 struct ip_address partner;
 
+#ifdef SMTPUTF8
+static stralloc idnhost = { 0 };
+static int      smtputf8 = 0; /*- if remote has SMTPUTF8 capability */
+static char    *enable_utf8 = 0; /*- enable utf8 */
+int             flagutf8;
+#endif
+
```

Comment:
These variables don't cost much to have, so we could lose this ifdef. Another function of the ifdef is to document that these are EAI-specific; we could accomplish that by putting them in a new `.c` file (`eai_remote.c`?) and linking with the new `.o`.